WrenchCL/Tools/WrenchLogger.py

```diff
@@ -527,11 +534,17 @@ def error(self, *args: Any, stack_info: Optional[bool] = True, compact: Optional
         self._log_with_color(self.ERROR_lvl, text, Color.RED if colorama_imported else None, stack_info, compact)
 
     def data(self, data: Any, object_name: Optional[str] = None, content: Optional[bool] = True, wrap_length: Optional[int] = None,
-             max_rows: Optional[int] = None, stack_info: Optional[bool] = False, indent: Optional[int] = 4) -> None:
+             max_rows: Optional[int] = None, stack_info: Optional[bool] = False, indent: Optional[int] = 4, truncate_values = True) -> None:
         """Logs a data message with optional formatting."""
+        try:
+            import pandas as pd
+        except ImportError:
+            pd = None
+        option_bu = pd.options
```

Comment:
AttributeError if pd is None.

Reply:
You're correct

![image](https://github.com/user-attachments/assets/27f1111e-7030-4a41-8808-34b19c06dda1)


made an update: [`19d325e` (#82)](https://github.com/WrenchAI/WrenchCL/pull/82/commits/19d325e856d4348306207046be03eb94bdc48efc)

that fixes it